packages/nextjs/components/blockexplorer/TransactionHash.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import Link from "next/link";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
+
+export const TransactionHash = ({ hash }: { hash?: string }) => {
```

Comment:
it seems that hash always exists, so it shouldn't be optional. And if it's optional `Link` will be broken